roles/installer/templates/statefulsets/postgres.yaml.j2

```
@@ -46,6 +46,27 @@ spec:
 {% endif %}
 {% if postgres_priority_class is defined %}
       priorityClassName: '{{ postgres_priority_class }}'
+{% endif %}
+{% if postgres_data_volume_init is defined and not is_openshift %}
```

Not tested at all on my side, but since `postgres_data_volume_init` has a default value, `is define` doesn't work as expected.

```suggestion
{% if postgres_data_volume_init and not is_openshift %}
```

---

Good catch, that is true.  I'll commit these changes and test it out with the awx-on-k3s scenario.  Thanks @kurokobo and @craph !